.github/workflows/deployment.yaml

```
@@ -189,6 +195,31 @@ jobs:
       environment: ${{ inputs.environment }}
       infra_commons_tag: ${{ needs.initChecks.outputs.infra_commons_tag_computed }}
 
+  invalidate_cache:
+      needs: [ deploy ]
+      if: ${{ inputs.run_invalidate_cache }}
+      runs-on: ubuntu-latest
+      environment: ${{ inputs.environment }}
+      steps:
+
+      - name: Checkout repository
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4
+        with:
+          aws-region: ${{ vars.AWS_REGION }}
+          role-to-assume: ${{ secrets.TERRAFORM_IAM_ROLE_ARN }}
+          role-session-name: interop-public-catalog-deployment-invalidate-cloudfront-distribution-cache-${{ github.run_number }}
+
+      - name: Invalidate CloudFront Cache
+        run: |
+          set -euo pipefail
+
+          aws cloudfront create-invalidation \
+            --distribution-id ${{ vars.CLOUDFRONT_DISTRIBUTION_ID }} \
+            --paths ${{ vars.CLOUDFRONT_INVALIDATION_PATHS }}
```

```suggestion
            --paths "${{ vars.CLOUDFRONT_INVALIDATION_PATHS }}"
```
does it support multiple comma-separated paths?

---

From the [documentation](https://docs.aws.amazon.com/cli/latest/reference/cloudfront/create-invalidation.html):
> --paths (string) The space-separated paths to be invalidated.

---

```suggestion
            --paths "${{ vars.CLOUDFRONT_INVALIDATION_PATHS }}"  #space separated
```
let's add a comment